doc/src/tutorial/printing.rst

```diff
@@ -231,8 +231,8 @@ printer.
 
 .. rubric:: Footnotes
 
-.. [#srepr-fn] SymPy does not use the Python builtin ``repr()`` function for
-   repr printing, because in Python ``str(list)`` calls ``repr()`` on the
+.. [#srepr-fn] SymPy does not use the Python builtin ``srepr()`` function for
```

Comment:
@asmeurer 
Does this footnote should have stayed unchanged? That means `srepr` back to `repr`.